### ISSUE TITLE
Remaining accessibility issues

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC-web-app#755]
## Home Page

- [x] In glossary, update span text on minimize buttons to say, collapse, or minimize, etc
- [x] Focus is lost when tabbing after the contact links
- [x]  Focus is lost off screen at the end of the page
- [x] Skip nav should focus on first element

## Registration and Reporting
- [x] Update alt tags in campaign guides to include all text seen in images
- [x] "Find by District" should be associated with the two fields either with a legend, or included in their aria labels
- [x] The first TD in each row in TBODY should be scope='row' so its clear to the screen reader that those fields belong to that document

## https://fec-dev-web.18f.gov/candidates
- [x] The first TD in each row in TBODY should be scope='row' so its clear to the screen reader that those fields belong to that candidate

## https://fec-dev-web.18f.gov/elections/
- [x] "Find by District" should be associated with the two fields either with a legend, or included in their aria labels

## https://fec-dev-web.18f.gov/filings
- [x] Run code sniffer on the filters, missing labels and duplicate ids
- [x] For radio buttons, label for only works with id's, need to use fieldset and legend to label radio buttons.
- [x] The first TD in each row in TBODY should be scope='row' so its clear to the screen reader that those fields belong to that document
- [x] Filters should be removed from tab order when hidden

## https://fec-dev-web.18f.gov/receipts
- [x] Run code sniffer on the filters, missing labels
- [x] For radio buttons, label for only works with id's, need to use fieldset and legend to label radio buttons.
- [x] The first TD in each row in TBODY should be scope='row' so its clear to the screen reader that those fields belong to that contributor

## https://fec-dev-web.18f.gov/disbursements
See above

## https://fec-dev-web.18f.gov/committee/
- [x] Tabs aren't keyboard accessible (includes sub tabs "compare by")
- [x] Select dropdown is missing label, see codesniffer
- [x] When tabbing to the end of the page, the style is "messed" up

# Remaining tasks:
- [x] Disabled links on the nave aren't contrast compliant
- [ ] "Help improve FEC" pop up, the close button isn't keyboard accessible
- [x] Code sniffer is throwing an error on the leaflet img tags missing an alt, if you can add a blank alt to those, that would be great
- [ ] year select should be labeled by neighboring text to give it more context

**Comment by noahmanger** [comment]
@jenniferthibault need some design help: the disabled links in the header still need to pass contrast requirements. Do you have any ideas how we should do this?

**Comment by jenniferthibault** [comment]
Ohhhh interesting! I was basing that choice on beta.consumerfinance, but it looks like their TBD content links aren't 508 compliant either.

What do you think about this?
 - Make text gray #737373 and 100% opacity for disabled links
 - Add parenthesis `(` x `)` around any disabled link to make it visually side-noted
 - Change `TBD` to `Content TBD` or `Page TBD` to make it more specific

That would look like this:
![screen shot 2015-09-25 at 11 21 15 am](https://cloud.githubusercontent.com/assets/11636908/10104431/95962916-6377-11e5-94ae-d7b2fdc35ce0.png)


If we REALLY want to do it up, we could add a tooltip explaining what `Page TBD` means, but that might be overkill. My gut says the above will be enough. Though obviously testing would be good here, I don't think 100% necessary to test that before launch given that we've heard from previous feedback that the rest of the page makes it clear.

**Comment by noahmanger** [comment]
I like it. Thanks.

**Comment by noahmanger** [comment]
Ok, got those fixed in a PR. That leaves the last three items on that list.

@jmcarp do you mind adding a little javascript to add alt-tags to the leaflet tiles? And if you have an idea for adding a `tabindex` to ethnio close button that should solve the problem of allowing that to be closed with the keyboard. But if not I think it's ok.

The last point about the year cycle select needing a label I take as more of a recommendation. The proper `aria-label` is there, and I believe we've seen in usability testing that people understand what it does.

**Comment by noahmanger** [comment]
Alt tags have been added to the tiles, and I made a separate issue for the Ethnio popup. I don't think we need to worrry about the year select, so I'm closing this out.
